packages/core/EventManager.ts

```diff
@@ -442,7 +441,7 @@ export default class EventManager {
       // Bookings should only have one calendar reference
       calendarReference = booking.references.filter((reference) => reference.type.includes("_calendar"))[0];
       if (!calendarReference) {
-        throw new Error("bookingRef");
+        return [];
```

Comment:
Instead of throwing an error if no calendar references, we just return an empty array